far/farlang.templ.m4

```diff
@@ -442,7 +442,7 @@ l:// Panels: functional keys
 "Pagalb"
 
 MF2
-"ПользМ"
+"Меню пользователя"
```

Comment:
These strings are short for a reason: they should fit and make some sense when the window isn't maximised.

Reply:
This, obviously, applies to all the changed F-key labels, not just this one.